### PR TITLE
Add option to display footer

### LINKED ISSUE #207: Add option to display footer specified by user
Add an option in the `gitprofile.config.js` titled **`footer`**.

```js
const config = {
  // ..............
  footer: 'Custom footer'
};
```

Use this new config in `src/components/GitProfile.jsx` to display the footer.

Please follow the checklist:

- [ ] The footer should be optional
- [ ] Update Readme
- [ ] Update prop definition
- [ ] Update type definition in `types/index.d.ts`

### PR DESCRIPTION
**Pull request by csarnataro**
Added an option in config file.
Added a component which renders the custom footer or a default footer if no custom footer is defined.
Updated readme file with related instructions.
Updated version to 2.2.2

Closes #207

### CODE REVIEW

**Comment by arifszn** [comment]
Nice. Will review it soon.

File: README.md [hunk 1]
```diff
@@ -71,7 +71,8 @@
 ✓ [Certification Section](#certifications)  
 ✓ [Education Section](#education)  
 ✓ [Projects Section](#projects)  
-✓ [Blog Posts Section](#blog-posts)
+✓ [Blog Posts Section](#blog-posts)  
+✓ [Custom footer](#custom-footer)
```
Comment (arifszn) [inline README.md:75]: Let's not declare it as a feature. We can just show it on the example config of `Customization`.

File: README.md [hunk 2]
```diff
@@ -617,6 +618,48 @@ module.exports = {
 
 The posts are fetched by [blog.js](https://github.com/arifszn/blog.js).
 
+
+### Custom footer (Optional)
```
Comment (arifszn) [inline README.md:622]: See the comment above. Let's remove this section.

File: gitprofile.config.js [hunk 3]
```diff
@@ -83,6 +83,10 @@ const config = {
     },
   ],
 
+  // Optional custom footer. See README file, section "custom-footer", for details
+  /*
+  footer: '[...]',
+  */
```
Comment (arifszn) [inline gitprofile.config.js:89]: There should not be any default footer. The user should be able to not display the footer if he chooses to.

**Review by arifszn — CHANGES_REQUESTED** [review]
Some small modifications are required. But good job indeed. And thanks a lot for the additional corrections.

File: gitprofile.config.js (hunk 3 above)
Comment (csarnataro) [reply gitprofile.config.js:89]: Just to be 100% sure: do you mean that if no footer is configured in the config file, then no footer will be rendered in the page? Correct?
And, if a user wants to show *"Made with GitProfile and ❤️"*, then they have to configure it explicitly?

Comment (arifszn) [reply gitprofile.config.js:89]: Yes, that's right.

**Comment by csarnataro** [comment]
Hi, I just updated the PR with requested changes.

**Comment by arifszn** [comment]
I haven't had the time to check the new changes. I will do it soon. Looks good anyway.

**Review by arifszn — APPROVED** [review]

**Pull request merged**